LICENCE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
We now have a `LICENSE` AND `LICENCE` file.

Reply:
Fixed.